src/copyright/ui/ajax-copyright-hist.php

```diff
@@ -100,10 +100,17 @@ function Output()
     else if($action=="update" || $action=="delete" || $action=="undo")
     {
       $id = GetParm("id", PARM_STRING);
-      list($upload, $item, $hash, $type) = explode(",", $id);
+      $getEachID = array_filter(explode(",", trim($id, ',')));
```

Comment:
The confusing behaviour can be prevented by adding an explicit callback
```suggestion
      $getEachID = array_filter(explode(",", trim($id, ',')), function($var) {
          return $var !== "";
      });
```
or maybe even something like:
```suggestion
      $getEachID = array_filter(explode(",", trim($id, ',')), function($var) {
          return ! is_numeric($var);
      });
```